crates/sui-config/src/node.rs

```diff
@@ -196,6 +196,12 @@ pub struct NodeConfig {
     #[serde(default = "bool_true")]
     pub enable_validator_tx_finalizer: bool,
 
+    /// Enable early transaction validation before submission to validator.
+    /// This checks for non-retriable errors (like old object versions) and rejects
+    /// transactions early to provide fast feedback to clients.
+    #[serde(default = "bool_true")]
+    pub enable_transaction_orchestrator_early_validation: bool,
```

Comment:
Can you add this to `transaction_driver_config`? Even though this is in Txn orchestrator, I think this might be moved to txn driver in future.